bitnami/opensearch/values.yaml

```diff
@@ -0,0 +1,2342 @@
+  ## ref: https://kubernetes.io/docs/user-guide/compute-resources/
+  ## We usually recommend not to specify default resources and to leave this as a conscious
+  ## choice for the user. This also increases chances charts run on environments with little
+  ## resources, such as Minikube. If you do want to specify resources, uncomment the following
+  ## lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  ## @param sysctlImage.resources.limits The resources limits for the container
+  ## @param sysctlImage.resources.requests The requested resources for the container
+  ##
+  resources:
+    ## Example:
+    ## limits:
+    ##    cpu: 100m
+    ##    memory: 128Mi
+    limits: {}
+    ## Examples:
+    ## requests:
+    ##    cpu: 100m
+    ##    memory: 128Mi
+    requests: {}
+
+## @section Opensearch Dashborads Parameters
+
+dashboards:
+  ## @param dashboards.enabled Enables Opensearch Dashboards deployment
+  ##
+  enabled: false
+  ## Bitnami Opensearch Dashboards image
+  ## @param dashboards.image.registry Opensearch Dashboards image registry
+  ## @param dashboards.image.repository Opensearch Dashboards image repository
+  ## @param dashboards.image.tag Opensearch Dashboards image tag (immutable tags are recommended)
+  ## @param dashboards.image.digest Opensearch Dashboards image digest in the way sha256:aa.... Please note this parameter, if set, will override the tag
+  ## @param dashboards.image.pullPolicy Opensearch Dashboards image pull policy
+  ## @param dashboards.image.pullSecrets Opensearch Dashboards image pull secrets
+  ## @param dashboards.image.debug Enable Opensearch Dashboards image debug mode
+  ##
+  image:
+    registry: docker.io
+    repository: bitnami/opensearch-dashboards
+    tag: 2.8.0-debian-11-r0
+    digest: ""
+    ## Specify a imagePullPolicy
+    ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+    ## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+    ##
+    pullPolicy: IfNotPresent
+    ## Optionally specify an array of imagePullSecrets.
+    ## Secrets must be manually created in the namespace.
+    ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+    ## e.g:
+    ## pullSecrets:
+    ##   - myRegistryKeySecretName
+    ##
+    pullSecrets: [ ]
+    ## Enable debug mode
+    ##
+    debug: false
+  ## Opensearch Dashboards service parameters
+  ##
+  service:
+    ## @param dashboards.service.type Opensearch Dashboards service type
+    ##
+    type: ClusterIP
+    ## @param dashboards.service.ports.http Opensearch Dashboards service web UI port
+    ##
+    ports:
+      http: 5601
+    ## Node ports to expose
+    ## @param dashboards.service.nodePorts.http Node port for web UI
+    ## @param dashboards.service.nodePorts.http-os Node port for REST API
+    ## NOTE: choose port between <30000-32767>
+    ##
+    nodePorts:
+      http: ""
+      http-os: ""
```

Comment:
This is not used

Reply:
[4608e3f](https://github.com/bitnami/charts/pull/17509/commits/4608e3f5761083f09783fc6c1d8e777ef1970a31)